packages/terra-show-hide/src/ShowHide.jsx

```diff
@@ -75,14 +115,31 @@ const ShowHide = (props) => {
     }
   }
 
+  const onBlur = () => {
+    if (containerIsActive) {
+      setContainerIsActive(false);
+    }
+  };
+
   return (
     <div {...customProps}>
       {!isOpen && preview}
-      <Toggle isOpen={isOpen}>
-        {children}
-      </Toggle>
+      <div
+        id={contentId}
```

Comment:
You should not need to specify an id attribute.

Reply:
The id here is needed for aria-controls attribute which is part of UXPLATFORM-9055:

https://github.com/cerner/terra-core/blob/67dcc73138ea81f7db11259eff5e3b952e955937/packages/terra-show-hide/src/ShowHide.jsx#L142